docs/get-started/features.md

```diff
@@ -61,7 +61,11 @@ The **cpu** feature source supports the following labels:
 | cpuid                   | &lt;cpuid flag&gt; | CPU capability is supported
 | hardware_multithreading |                    | Hardware multithreading, such as Intel HTT, enabled (number of logical CPUs is greater than physical CPUs)
 | power                   | sst_bf.enabled     | Intel SST-BF ([Intel Speed Select Technology][intel-sst] - Base frequency) enabled
-| [pstate][intel-pstate]  | turbo              | Set to 'true' if turbo frequencies are enabled in Intel pstate driver, set to 'false' if they have been disabled.
+| [pstate][intel-pstate]  | enabled            | Set to 'true' if the Intel pstate driver is in use, otherwise set to 'false'.
```

Comment:
Just thinking, would it be useful to advertise the status 🤔  I.e. pstate driver has been enabled if the label exists. The value of the label would indicate the status i.e. `active` or `passive`. The existence of the label could be used to check if enabled and the value for more detailed affinity rules. WDYT?

Reply:
Good idea. I will change the label to "status" with values active/passive based on the intel_pstate status.